windows/src/Defines.mak

```diff
@@ -221,3 +242,4 @@ SIGNCODE=@$(ROOT)\src\buildtools\signtime.bat signtool.exe $(SC_PFX_SHA1) $(SC_P
 #
 
 PLATFORM=Win32
```

Comment:
With this PR, is there a case for
`PLATFORM=Win64`?

Reply:
No, this is a specific workaround to a problem with msbuild as documented in the link given in the comment above. Nearly all Keyman projects are Win32, not Win64, so there is no case for adding `Win64` here.